zuul.d/pipelines.yaml

```diff
@@ -294,6 +294,9 @@
 
 - pipeline:
     name: third-party-check
+    # ⚠️  you should probably use 3pci-check instead of this pipeline
```

Comment:
yes, thanks. The main issue today is we need someone to add the 3pci github app to all the exiting 3pci projects listed in zuul tenant config. then we can remove this pipeline

Reply:
I added your comment in the comment block.